<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditTable</class>
 <widget class="QDialog" name="EditTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>编辑数据</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="BjTableView" name="TableView"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonNewItem">
         <property name="text">
          <string>新建条目</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonDelItem">
         <property name="text">
          <string>删除条目</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonOk">
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonCancel">
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BjTableView</class>
   <extends>QTableView</extends>
   <header>MainWindow/BjTableView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>ButtonOk</sender>
   <signal>clicked()</signal>
   <receiver>EditTable</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>302</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>330</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>EditTable</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>391</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>394</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
